src/main/java/edu/cornell/kfs/module/ar/document/service/impl/CuContractsGrantsInvoiceDocumentServiceImpl.java

```diff
@@ -67,6 +104,22 @@
         return templateParameters;
     }
     
+    protected String convertDateToString(Date dateValue) {
+        String dateValueAsString = StringUtils.EMPTY;
+        if (ObjectUtils.isNotNull(dateValue)) {
+            dateValueAsString = getDateTimeService().toDateString(dateValue);
+        }
+        return dateValueAsString;
+    }
+    
+    protected String convertNullStringToEmptyString(String stringValue) {
+        String valueToReturn = StringUtils.EMPTY;
+        if (ObjectUtils.isNotNull(stringValue)) {
+            valueToReturn = stringValue;
```

Comment:
Do we need to do this?  I think PdfFormattingMap should handle this stuff for us.

Reply:
I will remove and let future validation efforts confirm whether it needs to be added back in.